### ISSUE TITLE
Unobserved tasks of async method calls in Amazon sqs module

### ISSUE THREAD

**Issue by modabas** [dotnetcore/CAP#1029]
I have noticed there are some async method calls whose tasks are ignored. Such as:

```csharp
        public void Commit(object sender)
        {
            try
            {
                _sqsClient.DeleteMessageAsync(_queueUrl, (string)sender);
            }
            catch (InvalidIdFormatException ex)
            {
                InvalidIdFormatLog(ex.Message);
            }
        }
```

and 

```csharp
        public void Reject(object sender)
        {
            try
            {
                // Visible again in 3 seconds
                _sqsClient.ChangeMessageVisibilityAsync(_queueUrl, (string)sender, 3);
            }
            catch (MessageNotInflightException ex)
            {
                MessageNotInflightLog(ex.Message);
            }
        }
```

in [AmazonSQSConsumerClient.cs](https://github.com/dotnetcore/CAP/blob/master/src/DotNetCore.CAP.AmazonSQS/AmazonSQSConsumerClient.cs)

Any unobserved exception thrown from such calls will bubble up to .NET finalizer thread and may have the potential to eventually crash application.

So;

1. If the intention is to await return of these calls and catch exceptions, they may be modified with .GetAwaiter().GetResult()

2. Or if return of these async calls is not important and intention is to ignore them, a System.Task extension method may be used. Following code is from [Microsoft.Orleans](https://github.com/dotnet/orleans/blob/main/src/Orleans.Core.Abstractions/Utils/PublicOrleansTaskExtensions.cs) project. Also please check [some discussion about it](https://github.com/dotnet/orleans/issues/4731).

```csharp
    /// <summary>
    /// Utility functions for dealing with Tasks.
    /// </summary>
    public static class PublicOrleansTaskExtensions
    {
        private static readonly Action<Task> IgnoreTaskContinuation = t => { _ = t.Exception; };

        /// <summary>
        /// Observes and ignores a potential exception on a given Task.
        /// If a Task fails and throws an exception which is never observed, it will be caught by the .NET finalizer thread.
        /// This function awaits the given task and if the exception is thrown, it observes this exception and simply ignores it.
        /// This will prevent the escalation of this exception to the .NET finalizer thread.
        /// </summary>
        /// <param name="task">The task to be ignored.</param>
        [SuppressMessage("Microsoft.Performance", "CA1804:RemoveUnusedLocals", MessageId = "ignored")]
        public static void Ignore(this Task task)
        {
            if (task.IsCompleted)
            {
                _ = task.Exception;
            }
            else
            {
                task.ContinueWith(
                    IgnoreTaskContinuation,
                    CancellationToken.None,
                    TaskContinuationOptions.OnlyOnFaulted | TaskContinuationOptions.ExecuteSynchronously,
                    TaskScheduler.Default);
            }
        }
    }
```

3. Or a hybrid solution, catch some exceptions but ignore rest

```csharp
        public void Reject(object sender)
        { 
            Task.Run( async () =>
            {
                  try
                  {
                      // Visible again in 3 seconds
                      await _sqsClient.ChangeMessageVisibilityAsync(_queueUrl, (string)sender, 3);
                  }
                  catch (MessageNotInflightException ex)
                  {
                      MessageNotInflightLog(ex.Message);
                  }
            }).Ignore();
        }
```

Apart from this subject, there is a "RabbitMQ" string in [ITransport.AmazonSQS.cs](https://github.com/dotnetcore/CAP/blob/master/src/DotNetCore.CAP.AmazonSQS/ITransport.AmazonSQS.cs)

```csharp
        public BrokerAddress BrokerAddress => new BrokerAddress("RabbitMQ", string.Empty);
```

Regards,

**Comment by yang-xiaodong** [comment]
Thank you for your feedback.

Yes, you are right. Would you like to make a PR for this?

**Comment by yang-xiaodong** [comment]
Fixed in version `5.2.0-preview-152861792`
